.husky/scripts/dont-trust.sh

```
@@ -0,0 +1,43 @@
+    # Check if commit signing is enabled
+    if [ "$(git config --get commit.gpgsign)" != "true" ]; then
+        echo "❌ Error: Commit signing not enabled"
+        echo "Run: git config --global commit.gpgsign true"
+        exit 1
+    fi
+
+    # https://git-scm.com/docs/git-log#Documentation/git-log.txt-emGem
+    # Get detailed signing information
+    VERIFY_SIGNING=$(git log -1 --pretty=%G?)
+
+    if [[ "$VERIFY_SIGNING" != 'G']]; then
```

Shellcheck (0.10.0)

[error] Couldn't parse this test expression. Fix to allow more checks.

(SC1073)